CONTRIBUTING.md

```diff
@@ -29,25 +47,21 @@ Before you begin, perform initial setup:
 
         $ ./Utilities/SetupForDevelopment.sh
 
-    This will prompt for your GitLab user name and configure a remote
-    called `gitlab` to refer to it.
-
 5. (Optional but highly recommended.)
+    Install the official [GitHub CLI utility] on your developing station. This
+    tool greatly simplifies interacting with GitHub from the command line.
+
+6. (Optional but highly recommended.)
     [Register with the Viskores dashboard] on Kitware's CDash instance to
     better know how your code performs in regression tests. After
     registering and signing in, click on "All Dashboards" link in the upper
     left corner, scroll down and click "Subscribe to this project" on the
     right of Viskores.
 
-6.  (Optional but highly recommended.)
-    [Sign up for the Viskores mailing list] to communicate with other
+7.  (Optional but highly recommended.)
+    [Join the Viskores conversation] to communicate with other
```

Comment:
Why use `conversation` instead of `discussions`? GitHub calls the tab discussions. I believe conversations refer to something else (i.e., the responses to comments like this.)

Reply:
Good catch! It makes sense, find this addressed in the latest git push